src/controls.css

```diff
@@ -0,0 +1,38 @@
+.controls-title {
+  font-size: 40px;
+  margin-bottom: 20px;
+}
+
+.controls ol {
```

Comment:
You are now using ul not ol in html. That's why instead of using element names we use classnames.

<ul className="instructions-list">
</ul>

.instructions-list {}

No need to update css when you change say ul to ol.